doc/README.md

```diff
@@ -2,47 +2,42 @@ Kivy - Documentation
 ====================
 
 You can access the latest documentation on the web:
-  * http://kivy.org/docs
 
+* http://kivy.org/docs
 
-How to build the documentation
-------------------------------
 
-You need to install:
+Install Sphinx
+--------------
 
-  * Python Sphinx
-    - With pip:
-    ```
-	pip install sphinx
-	```
+- With pip:
+  
+  ``pip install sphinx``
 
-    - With apt-get:
-    ```
-    apt-get install python-sphinx
-    ```
+- With apt-get:
+    
+  ``apt-get install python-sphinx``
 
-    - With MacPorts:
-    ```
-	port install py34-sphinx
-	```
+- With MacPorts:
+  
+  ``port install py34-sphinx``
 
-    - On Windows (or from inside your virtualenv):
+- On Windows (or from inside your virtualenv):
 
-          Get pip (https://pypi.python.org/pypi/pip). You'll use it to install the dependencies.
+  Get pip (https://pypi.python.org/pypi/pip). You'll use it to install the dependencies.
 
-          To install pip, run python setup.py install in the pip directory. Now run:
-          ```
-          pip install sphinxcontrib-blockdiag sphinxcontrib-seqdiag
-          pip install sphinxcontrib-actdiag sphinxcontrib-nwdiag
-          ```
-          Or just use the provided `doc-requirements.txt`:
-          ```
-          pip install -r doc-requirements.txt
-          ```
+  To install pip, run ``python setup.py install`` in the pip directory. Now run:
+    
+  ``pip install sphinxcontrib-blockdiag sphinxcontrib-seqdiag``
+  
+  ``pip install sphinxcontrib-actdiag sphinxcontrib-nwdiag``
+    
+  Or just use the provided *doc-requirements.txt*:
+    
+  ``pip install -r doc-requirements.txt``
+  
+Building the documentation
+--------------------------
 
-  * Latest kivy
```

Comment:
This part is essential when building the docs when a user wants to contribute, otherwise it may result in outdated parts or even collisions.